index.js

```diff
@@ -46,12 +51,19 @@ class RC433EtekcitySwitch {
       .getCharacteristic(Characteristic.On)
       .on('set', (value, callback) => {
         state = value;
+        var signal;
```

Comment:
what do you think of using `let` here as well?
// edit: sorry, ich war im Arbeitsmodus, Rest mach ich in Deutsch :)